starboard/nplb/posix_compliance/posix_file_close_test.cc

```diff
@@ -23,8 +23,17 @@ namespace nplb {
 namespace {
 
 TEST(PosixFileCloseTest, CloseInvalidFails) {
-  int fd = -1;
-  EXPECT_FALSE(close(fd) == 0);
+  const int fd = -1;
+
+  const int result = close(fd);
+  EXPECT_EQ(result, -1)
```

Comment:
Reading the close() documentation, it says two things: 1) it doesn't return 0 and 2) it does return -1. So I'd suggest here:
```
EXPECT_NE(result, 0) << "close() with an invalid fd should not succeed";
constexpr int kExpectedFailure = -1;
EXPECT_EQ(result, kExpectedFailure) << "close() with an invalid fd should "
    << "return << kExpectedFailure << " but returned " << result;
```

WDYT?
While the current expectation encompasses both, the error messages are
different (this is already done in e.g. posix_file_fsync_test.cc below)